mrp_food/migrations/12.0.3.0.0/pre-migration.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (C) 2024 - Today: GRAP (http://www.grap.coop)
+# @author: Quentin Dupont (quentin.dupont@grap.coop)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+
+    # create new tags from seasonality
+    openupgrade.logged_query(
+        env.cr,
+        """
+        INSERT INTO mrp_bom_tag(company_id,name,complete_name)
```

Comment:
This model is defined in grap module ``mrp_bom_tag`` but I don't see a dependency to that module in ``mrp_food``.  It should be set don't you think ?

Reply:
Bien vu !
Je fais la modif mais faut que je reteste tout ça avec notre base, je fais ça asap ! ya eu bcp de changements de partout mais pour du mieux c'est chouettos !